modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormRendererImpl.java

```diff
@@ -45,6 +45,36 @@
 @Component(immediate = true, service = DDMFormRenderer.class)
 public class DDMFormRendererImpl implements DDMFormRenderer {
 
+	@Override
+	public Map<String, Object> getContext(
+			DDMForm ddmForm, DDMFormLayout ddmFormLayout,
+			DDMFormRenderingContext ddmFormRenderingContext)
+		throws Exception {
+
+		Map<String, Object> ddmFormTemplateContext =
+			_ddmFormTemplateContextFactory.create(
+				ddmForm, ddmFormLayout, ddmFormRenderingContext);
+
+		ddmFormTemplateContext.put("editable", false);
+
+		ddmFormTemplateContext.remove("fieldTypes");
```

Comment:
hey @natocesarrego will this be removed only for the context of the user preview?

Reply:
@matuzalemsteles, this is not a new code, actually, the method having it was just moved in the class. In addition, it'll be removed from the context of all places triggering the Form rendering. It's important to emphasize that this `context` is, on a React perspective, the `props` passed to the Form React component.

@matuzalemsteles, do you see any problem to remove the `fieldTypes` from the context?

Reply:
@natocesarrego I see, we're using this information in the Forms builder, I'm not sure if it will affect it there.